settings.py

```diff
@@ -24,5 +24,9 @@
 USE_TZ = False
```

Comment:
The `USE_TZ` setting is set to `False`, which means Django will store naive datetime objects in the database. Consider setting `USE_TZ = True` to enable timezone support and avoid potential issues with datetime objects.